apps/frontend/src/pages/people/index.tsx

```diff
@@ -30,7 +30,7 @@ const PeoplePage: React.FC = () => {
     };
 
     fetchConnections();
-  }, []);
+  }, [connections]);
```

Comment:
NOTE: I was having issues figuring out how to refresh the frontend component after the WS message is received and processed. As the value that's changing is connections, making `useEffect` dependent on the value leads to the desired refresh behavior. I'm open to other refresh approaches, but this one worked.